content/test/integration/syncronization/failed-deployments.spec.ts

```diff
@@ -20,11 +19,16 @@ loadTestEnvironment()('Errors during sync', (testEnv) => {
       // Start server1
       await this.server1.startProgram()
 
-      this.accessCheckerStub1 = stub(this.server1.components.accessChecker, "hasAccess");
-      this.accessCheckerStub2 = stub(this.server2.components.accessChecker, "hasAccess");
-      this.accessCheckerStub1.returns(Promise.resolve([]));
-      this.accessCheckerStub2.onCall(0).returns(Promise.resolve(['anyError']));
-      this.accessCheckerStub2.onCall(1).returns(Promise.resolve([]));
+      this.validatorStub1 = stub(this.server1.components.validator, 'validate')
+      this.validatorStub1.returns(Promise.resolve({ ok: true }))
+      this.validatorStub2 = stub(this.server2.components.validator, 'validate')
+      this.validatorStub2.returns(Promise.resolve({ ok: true }))
+
+      this.serverValidatorStub2 = stub(this.server2.components.serverValidator, 'validate')
+      this.serverValidatorStub2.onCall(0).returns(Promise.resolve({ ok: false, message: 'anyError' }))
+      this.serverValidatorStub2.onCall(1).returns(Promise.resolve({ ok: true }))
+      this.serverValidatorStub2.onCall(2).returns(Promise.resolve({ ok: true }))
+      this.serverValidatorStub2.onCall(3).returns(Promise.resolve({ ok: true }))
```

Comment:
It would be interesting to come up with a fancy way to do this.. I had to do the same many times

Reply:
will check if can be replaced with specific calls with arguments o a mocked impl

Reply:
Oh I didn't mean to ask for a change, it was simply a comment